src/support/z_file_locate.erl

```diff
@@ -214,6 +214,7 @@ part_file(Filename, Opts) ->
             Error
     end.
 
+-spec render(#module_index{}, #context{}) -> #part_data{}.
```

Comment:
The spec in line 217 uses a record, please define a type for the record and use that instead.